src/anomalib/data/base/video.py

```diff
@@ -153,6 +153,7 @@ def __getitem__(self, index: int) -> dict[str, str | torch.Tensor]:
             msg = "self.indexer must be an instance of ClipsIndexer."
             raise TypeError(msg)
         item = self.indexer.get_item(index)
+        item["image"] = item["image"].to(torch.float32)
```

Comment:
I would suggest to use `torchvision.transforms.v2.functional.to_dtype_video` here, with `scale=True`. This way we ensure that the inputs are not only of the right data type, but also scaled between [0, 1] instead of [0, 255].

Reply:
Perfect, already done.